functions.php

```diff
@@ -110,4 +110,23 @@ function currentFilter(string $category):string {
         <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
         <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
     }
-}
\ No newline at end of file
+}
+
+/**
+ * inputValidation function takes string$item, string$category, int$price, int$remaining (all should come from $_GET) and return either a INSERT query or error message.
```

Comment:
specifies what the function takes but not what it does with it.

Reply:
Agreed